lib/src/json_schema/validator.dart

```diff
@@ -661,6 +714,40 @@ class Validator {
     return false;
   }
 
+  //////
+  // Helper functions to deal with unevaluatedItems.
+  //////
+  _createUnevaluatedItemsContext() {
```

Comment:
:nit: If we're using the list as a stack, maybe `push` and `pop` would make more sense than `create` and `drop`?